lib/compiler_rt/mulXf3.zig

```diff
@@ -5,8 +5,32 @@
 const std = @import("std");
 const math = std.math;
 const builtin = @import("builtin");
-const compiler_rt = @import("../compiler_rt.zig");
+const arch = builtin.cpu.arch;
+const is_test = builtin.is_test;
+const linkage: std.builtin.GlobalLinkage = if (builtin.is_test) .Internal else .Weak;
+pub const panic = @import("common.zig").panic;
+
+comptime {
+    @export(__mulsf3, .{ .name = "__mulsf3", .linkage = linkage });
+    @export(__muldf3, .{ .name = "__muldf3", .linkage = linkage });
+    @export(__mulxf3, .{ .name = "__mulxf3", .linkage = linkage });
+    @export(__multf3, .{ .name = "__multf3", .linkage = linkage });
```

Comment:
do we want these also in separate compilation units? looks like right now these will all go into mulXf3.o.

Reply:
I think that's OK. It's only 4 symbols that share functionality just accept different operand types. I actually think we should split out `addXf3.zig` into `addXf3.zig` and `subXf3.zig`, and combine `divsf3.zig`, `divtf3.zig`, `divdf3.zig`, `divxf3.zig`, and `divti3.zig` into `divXf3.zig`. Thoughts?

Reply:
I think the different types are especially useful to keep separate because typically a target will have support for one operand type but not another. For example x86_64 is going to need functions with f128 operand types, but it will not need any functions with f32 or f64 operand types. So putting f32, f64, f128 all into the same compilation unit is going to create garbage.